lib/configs/vue.ts

```diff
@@ -129,6 +129,8 @@ export function vue(options: ConfigOptions): Linter.Config[] {
 				'vue/new-line-between-multi-line-property': 'error',
 				// Add consistent padding between blocks
 				'vue/padding-line-between-blocks': 'error',
+				// prefer separated static and dynamic class attributes
+				'vue/prefer-separate-static-class': 'error',
```

Comment:
Or 
```suggestion
				'vue/prefer-separate-static-class': 'error',
```

to not explain that `prefer-separate-static-class` is for `prefer separated static and dynamic class` 👀